api/external/infra_proxy/response/servers.proto

```diff
@@ -4,30 +4,39 @@ package chef.automate.api.infra_proxy.response;
 option go_package = "github.com/chef/automate/api/external/infra_proxy/response";
 
 message CreateServer {
+    // Chef Infra Server.
     Server server = 1;
 }
 
 message DeleteServer {
+    // Chef Infra Server.
     Server server = 1;
 }
 
 message UpdateServer {
+    // Chef Infra Server.
     Server server = 1;
 }
 
 message GetServers {
+    // Chef Infra Servers.
```

Comment:
Suggest going with this for consistency with, e.g. https://github.com/chef/automate/blob/6f7008e271ad82b051805b49a316babdfb21529e/api/interservice/infra_proxy/response/environments.proto#L25 or https://github.com/chef/automate/blob/099a38e60c4617ca0d54fc358bc46b11fcca8653/components/automate-gateway/api/iam/v2/common/policy.proto#L24
```suggestion
    // List of Chef Infra Servers.
```

This same comment applies wherever you have servers (plural) in the protos.